File: JAVA/Exercicios/src/exercicios_java/Ex18.java
package exercicios_java;

import java.util.Scanner;

public class Ex18 {

	public static void main(String[] args) {
		//  Criar um programa para analisar a velocidade de um automóvel. 
		// Solicitar via teclado os valores da aceleração (a em m/s2), velocidade inicial (v0 em m/s) e o tempo de percurso (t em s). 
		// Calcular e exibir a velocidade final do automóvel em km/h. E exibir mensagem de acordo com a tabela abaixo:

		Scanner ler = new Scanner (System.in);
		
		int a, v1, t, v;
		
		System.out.printf("Digite o valor da aceleração: ");
		a = ler.nextInt();
		
		System.out.printf("Digite o valor da velocidade inicial: ");
		v1 = ler.nextInt();
		
		System.out.printf("Digite o valor da tempo de percurso: ");
		t = ler.nextInt();
		
		v = v1 + (a*t);
		
		if (v <=40) {
			System.out.printf("Veículo muito lento, velocidade de %d", v);
		} else if (v > 40 && v <= 60) {
			System.out.printf("Velocidade permitida, velocidade de %d", v);
		} else if (v>60 && v <=80) {
			System.out.printf("Velocidade de cruzeiro, velocidade de %d", v);
		} else if (v > 80 && v <= 120) {
			System.out.printf("Veículo rápido, velocidade de %d", v);
		} else {
			System.out.printf("Veículo muito rápido, velocidade de %d", v);
		}
		
	
		
		
		
	}

}

File: Python/Exercicios/Ex18.py
#Criar um programa para analisar a velocidade de um automóvel. Solicitar via teclado os valores da aceleração (a em m/s2), velocidade inicial (v0 em m/s) e o tempo de percurso (t em s). Calcular e exibir a velocidade final do automóvel em km/h. E exibir mensagem de acordo com a tabela abaixo:

a = float(input('Digite a aceleração: '))
v1 = float(input('Digite o valor da velocidade inicial: '))
t = float(input('Digite o tempo do percurso: '))

vf = (v1 + (a*t)) * 36/10  

if (vf <=40):
    print (f'O veículo está muito lento, velocidade: {vf:.2f} km/h')
elif (vf>40) and (vf<=60):
    print (f'O veículo está na velocidade permitida, velocidade: {vf:.2f} km/h')
elif (vf>60) and (vf <=80):
    print (f'O veículo está com velocidade de cruzeiro, velocidade: {vf:.2f} km/h')
elif (vf >80) and (vf <=120):
    print (f'O veículo está rápido, velocidade: {vf:.2f} km/h')
else: 
    print (f'O veículo está muito rápido, velocidade: {vf:.2f} km/h')